src/promptflow-tools/tests/test_aoai.py

```diff
@@ -15,21 +15,21 @@ def test_aoai_completion(self, aoai_provider):
         # test whether tool can handle param "stop" with value empty list
         # as openai raises "[] is not valid under any of the given schemas - 'stop'"
         aoai_provider.completion(
-            prompt=prompt_template, deployment_name="text-ada-001", stop=[], logit_bias={}
+            prompt=prompt_template, deployment_name="gpt-4", stop=[], logit_bias={}
```

Comment:
Why need to use `gpt-4` rather than `gpt-35`?

Reply:
only gpt-4 and gpt-35-turbo support completion api， but this resource doesn't have enough quota when create gpt-35-turbo deployment
![image](https://github.com/microsoft/promptflow/assets/75061414/94333664-446d-47d8-8d59-c35c8658e0e8)

Reply:
create new resource on region sweden-central [promptflow-ci-sweden-central - Microsoft Azure](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/chesi-sweden-central/providers/Microsoft.CognitiveServices/accounts/promptflow-ci-sweden-central/oaideployments) which supports all models used in tool tests